### ISSUE TITLE
[@uifabric/example-app-base] Focus Indicator is not visible on the "Show Code" button in high contrast mode.

### ISSUE THREAD

**Issue by lucas-stanford** [microsoft/fluentui#18051]
### Environment Information
OS Build: Version 20H2(19042.867)
Browser: Version 89.0.4389.90 (Official Build) (64-bit)

### Describe the issue:
If focus is not visible on any control in high contrast black mode , then it can confuse and disorient the user navigate in high contrast black mode while TAB navigation through keyboard. 

### Please provide a reproduction of the issue in a codepen:
Repro'd here: https://fabricweb.z5.web.core.windows.net/oufr/8.14.1/#/examples/activityitem

#### Actual behavior:
In high contrast mode, Focus indicator is not visible on the "Export to code pen" and "Show Code" button.

#### Expected behavior:
In high contrast mode, Focus indicator should be visible on the "Export to code pen" and "Show Code" button.

### Documentation describing expected behavior
n/a

![nohighlighting](https://user-images.githubusercontent.com/3011776/117091915-a6d5a280-ad11-11eb-990c-13650ebf2b4b.gif)

**Comment by tyveille** [comment]
@TristanWatanabe 

Any update on this issue?
